br/use-br-command-line-tool.md

```diff
@@ -341,6 +341,8 @@ br restore full \
 
 如果需要恢复的数据并不是存储在本地，而是在 Amazon 的 S3 后端，那么需要在 `storage` 子命令中指定 S3 的存储路径，并且赋予 BR 节点和 TiKV 节点访问 Amazon S3 的权限。
 
+完成一次备份恢复需要的权限包括 `s3:ListBucket`, `s3:PutObject`，`s3:GetObject`。
```

Comment:
Please update https://docs.pingcap.com/zh/tidb/stable/use-br-command-line-tool#%E5%A4%87%E4%BB%BD%E6%95%B0%E6%8D%AE%E5%88%B0-amazon-s3-%E5%90%8E%E7%AB%AF%E5%AD%98%E5%82%A8 too.

Also, could you add documentation of what permission TiKV needs?

Reply:
This PR already update in the place that th link points to ... maybe your given link is wrong?

Reply:
Done, PTAL again